multisig/proposals/arbitrumsepolia/vip-009/index.ts

```diff
@@ -0,0 +1,22 @@
+import { makeProposal } from "../../../../src/utils";
+
+export const XVS_VAULT_PROXY = "0x407507DC2809D3aa31D54EcA3BEde5C5c4C8A17F";
+export const REWARDS_SPEED = "31709791983764";
+export const XVS = "0x877Dc896e7b13096D3827872e396927BbE704407";
+
+const vip009 = () => {
+  return makeProposal([
+    {
```

Comment:
We have to transfer the XVS tokens from the Treasury to the XVSStore first, as we made [here](https://github.com/VenusProtocol/vips/blob/develop/multisig/proposals/ethereum/vip-006/index.ts#L37-L41) for Ethereum, for example

Reply:
[Done](https://github.com/VenusProtocol/vips/pull/274/commits/2625785232a74a99b08a003d5155129fb39b5cf5)